ocp.tf

```diff
@@ -7,7 +7,7 @@ provider "ibm" {
 resource "random_id" "label" {
   count       = var.cluster_id == "" ? 1 : 0
   byte_length = "2" # Since we use the hex, the word lenght would double
```

Comment:
can you pls also fix the type: lenght ..

Reply:
Done.